src/artemis/stores/mapgraph/read.cljs

```diff
@@ -73,7 +73,13 @@
               (assoc result k val)
 
               (ref? store val)
-              (assoc result k (pull store join-expr val gql-context))
+              (assoc result k (pull store
+                                    (ref-join-expr
+                                     join-expr
+                                     (some-> entity (get expr) first namespace)
```

Comment:
When you've got something like `{:id 123 :nested [:object/id 456]}`, this will get `"object"`. Is it too brittle way of doing it?

Reply:
seems fine to me